### PR TITLE
Input field for "Reason for Red"

### LINKED ISSUE #197: Reason for Amber/Red
When Red or Amber is selected - the note field 'Reason for Red' or 'Reason for Amber' should appear and be blank. The user should not have to add a note to get that to appear.

### PR DESCRIPTION
**Pull request by MykytaKiriushyn**
Input field for "Reason for Red"
Fixes #197

### CODE REVIEW

File: o-fish-ios/Views/ReportFlow/Risk/RiskView.swift [hunk 1]
```diff
@@ -35,6 +35,10 @@ struct RiskView: View {
                         InputField(title: "Reason for amber", text: self.$report.inspection.summary.safetyLevel.amberReason)
                     }
 
+                    if self.report.inspection.summary.safetyLevel.level == .red {
+                        InputField(title: "Reason for red", text: self.$report.inspection.summary.safetyLevel.redReason)
```
Comment (akaktus) [inline o-fish-ios/Views/ReportFlow/Risk/RiskView.swift:39]: Need add this key in localization "Reason for red"

**Review by am-MongoDB — CHANGES_REQUESTED** [review]
The comment from Artem needs to be addressed.

Changes to the model need to be coordinated between iOS, Android, web, Realm and the existing documents in Atlas so that Realm Sync continues to work.

Rather than having `redReason` and `amberReason`, it would be better to have a single field called `reason`.

As a stop gap, could just use `amberReason` for both red and amber *but* need to keep open an issue to rename the field in the schema.

**Review by akaktus — APPROVED** [review]
LGTM

**Comment by am-MongoDB** [comment]
@MykytaKiriushyn Note that this is still waiting for the review comments to be addressed

**Review by am-MongoDB — APPROVED** [review]

**Pull request merged**